packages/location-management/src/ducks/location-units.ts

```diff
@@ -61,7 +61,7 @@ export interface LocationUnitPayloadPOST {
   type: string;
   syncStatus: string;
   properties: Properties;
-  locationTags: LocationUnitTag[];
+  locationGroups: LocationUnitGroup[];
```

Comment:
we are receiving locationTags inside payload how will we deal with that over here ?

Reply:
In that case we don't have any actions in this and all the locations, we will handle that while sending data to backend or receiving data, we change the key for ui

Reply:
yes you dont need to modify these keys

Reply:
@dubdabasoduba regarding these keys,
After discussing with @ashfahan, need your input here .
Should I replace `locationTags` key with `locationGroups` ? it is a part of payload.